Mathlib/NumberTheory/NumberField/InfinitePlace/Basic.lean

```diff
@@ -0,0 +1,498 @@
+/-
+Copyright (c) 2022 Xavier Roblot. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Alex J. Best, Xavier Roblot
+-/
+import Mathlib.NumberTheory.NumberField.InfinitePlace.Embeddings
+import Mathlib.NumberTheory.NumberField.Norm
+import Mathlib.RingTheory.RootsOfUnity.PrimitiveRoots
+import Mathlib.Topology.Instances.Complex
+
+/-!
+# Infinite places of a number field
+
+This file defines the infinite places of a number field.
+
+## Main Definitions and Results
+
+* `NumberField.InfinitePlace`: the type of infinite places of a number field `K`.
+* `NumberField.InfinitePlace.mk_eq_iff`: two complex embeddings define the same infinite place iff
+  they are equal or complex conjugates.
+* `NumberField.InfinitePlace.IsReal`: an infinite place is real if it is defined by a
+  real embedding.
```

Comment:
```suggestion
* `NumberField.InfinitePlace.IsReal`: The predicate on infinite places saying
  that a place is real, i.e., defined by a real embedding.
```
I think that right now it's easy to misinterpret the docstring as saying that `IsReal` is a theorem.